microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/MetricsChecker.java

```diff
@@ -66,8 +66,8 @@ public MetricsChecker validateRetryCallsSucceededTotal(int callsSucceededTotal)
      */
     public MetricsChecker validateRetryCallsSucceededNotTriedTotal(int callsSucceededNotTriedTotal) {
         response.body(
-                "application.'ft." + FailSafeDummyService.class.getName()
-                        + ".isReady.retry.callsSucceededNotRetried.total'",
+                "base.'ft.retry.calls.total;method=" + FailSafeDummyService.class.getName()
```

Comment:
Honestly, I can't say I understand this change, but definitely you should update also the javadoc for this method and the other one too.

Reply:
For any reviewer, the reason of these changes lies at Fault Tolerance changes https://github.com/eclipse/microprofile-fault-tolerance/blob/master/spec/src/main/asciidoc/release_notes.asciidoc#release-notes-for-microprofile-fault-tolerance-30

Reply:
> In addition, metrics have moved from the application: scope to the base: scope for consistency with other MicroProfile specifications.

Looking at this more.. I don't understand why all other methods here don't require the similar changes too due to the FT changes, this seems wrong

Reply:
I see, all tests using the other methods are ignored :facepalm: 
@xstefank would you mind to deal with unignoring these tests and updating metrics checks for them too?

Reply:
@jbliznak I will try to figure this out. Thanks for the investigation.